handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -184,8 +188,16 @@
                 EXTRA_SUPPORTED_TLS_1_3_CIPHERS = new String [] { "TLS_AES_128_GCM_SHA256",
                         "TLS_AES_256_GCM_SHA384" ,
                         "TLS_CHACHA20_POLY1305_SHA256" };
+
+                StringBuilder ciphersBuilder = new StringBuilder(64);
```

Comment:
Current result will be 75 bytes. Consider starting with that number

Reply:
I think I will just use 128 to be safe... as this is not expected to be called very often this is good enough.